server/testutil/webtester/webtester.go

```diff
@@ -440,7 +441,26 @@ func GetBazelBuildFlags(wt *WebTester, appBaseURL string, opts ...SetupPageOptio
 	for _, fn := range opts {
 		fn(wt)
 	}
-	rawBazelrc := wt.Find(`[data-header=".bazelrc"] .contents`).Text()
+
+	refetch := true
+	attempts := 1
+	var rawBazelrc string
+	for refetch && attempts <= 5 {
+		rawBazelrc = wt.Find(`[data-header=".bazelrc"] .contents`).Text()
```

Comment:
Since this is actually an async call on the page, you can use the builtin `wt.driver.WaitWithTimeout` to wait for this element to exist